provisioner/k8s/README.md

```diff
@@ -1,3 +1,112 @@
-# k8s
+# k8s Provisioner
 
-A concrete Provisioner implementation for "plain" k8s bundles.
+## Summary
+
+The k8s provisioner is a core rukpak provisioner that knows how to interact with `k8s` format bundles. 
+The `k8s` bundles, or plain bundles, are simply container images containing a set of static kubernetes YAML manifests in a given directory.  
+For example, below is an example Dockerfile that builds a plain `k8s` bundle from a directory containing static manifests.  
+```dockerfile
+FROM scratch
+LABEL manifests="/manifests"
+COPY ./manifests /manifests
+```
```

Comment:
I think this example is a bit more clearer if we showed off the local directory structure (e.g. using `tree ./manifests`) so the user gets an idea on what we're copying in the container. Any thoughts? This can be done in a follow-up if we think that it's clearer.

Reply:
That makes sense, I think we can include that. What we really need is a plain bundle spec to link to, some documentation that discusses this bundle format. The k8s provisioner and the plain bundle spec are related but different, in the doc we are conflating the two because there is no plain bundle format doc yet

Reply:
Yep completely agree.

Reply:
I've added a tree example to this section, but will make another pass once we merge the plain+v0 bundle spec documentation to the repository so we can link to there